src/mobile/src/main/java/tupperdate/api/ImageApi.kt

```diff
@@ -0,0 +1,44 @@
+package tupperdate.api
+
+import android.net.Uri
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.FileProvider
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import java.io.File
+
+interface ImageApi {
+    fun launch()
```

Comment:
I would maybe name this `pick()` or something that better explains what this is about.